experimental/trusted_shuffler/server/src/http.rs

```diff
@@ -88,33 +131,63 @@ fn trusted_shuffler_to_hyper_request(
     trusted_shuffler_request: TrustedShufflerRequest,
 ) -> Request<Body> {
     let body = Body::from(trusted_shuffler_request.body);
-    Request::builder()
+
+    let mut request = Request::builder()
         .uri(trusted_shuffler_request.uri)
         .method(Method::POST)
         .version(http::Version::HTTP_2)
         .body(body)
-        .expect("Failed to convert Trusted to Hyper Request")
+        .expect("Failed to convert Trusted to Hyper Request");
+
+    let headers = request.headers_mut();
```

Comment:
I'm surprised this does not need to be mut, is that because it is used by value below?

Reply:
I think it is fine: `headers: &mut HeaderMap`.